installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,122 @@
+====
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[manually create and maintain IAM credentials].
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+[id="installation-requirements-user-infra_{context}"]
+== Requirements for a cluster with installer-provisioned infrastructure
+
+For a cluster that contains installer-provisioned infrastructure, the Machine API will be responsible for creating the worker nodes declared by the MachineSets available on `<install-dir>/openshift/99_openshift-cluster-api_worker-machineset-*.yaml`
+
+This section describes the requirements for deploying {product-title} on an existing VPC with Local Zone subnets.
+
+include::modules/installation-machine-requirements.adoc[leveloffset=+2]
+include::modules/installation-minimum-resource-requirements.adoc[leveloffset=+2]
+include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
+include::modules/installation-aws-arm-tested-machine-types.adoc[leveloffset=+2]
+
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+1]
+
+include::modules/installation-aws-marketplace-subscribe.adoc[leveloffset=+1]
+
+//These next four modules seem like they're placed awkwardly, but you need the subnets to put in the install-config.yaml file.
+include::modules/installation-creating-aws-vpc-localzone.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-vpc-localzone.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+include::modules/installation-creating-aws-subnet-localzone.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+2]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
+
+include::modules/installation-generate-aws-user-infra-install-config.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[Configuration and credential file settings] in the AWS documentation for more information about AWS profile and credential configuration.
+
+//include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+//Put this back if QE validates it.
+
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+2]
+
+include::modules/installation-extracting-infraid.adoc[leveloffset=+1]
+
```

Comment:
After set all the manifests, we can create the cluster. I am not sure if there is a module for that, but it's a single command:
```
./openshift-install create cluster --log-level=debug
```

https://github.com/mtulio/mtulio.labs/blob/article-ocp-aws-lz/docs/articles/ocp-aws-local-zones-day-0.md#install-the-cluster-